amr-wind/diffusion/incflo_diffusion.cpp

```diff
@@ -20,6 +20,7 @@ Vector<Array<LinOpBCType, AMREX_SPACEDIM>> get_diffuse_tensor_bc(
             switch (bc) {
             case BC::pressure_inflow:
             case BC::pressure_outflow:
+            case BC::mass_inflow_outflow:
```

Comment:
I'm guessing you had to pick Neumann out of the two options, for the sake of the possibility of outflow -- is there any concern of inaccuracy when the inflow is active?

Reply:
I think we should be doing Dirichlet as we have the "correct" velocities in the boundary registers for both inflow and outflow, let me confirm.

Reply:
Ok good, I'll let you confirm. I wasn't sure how you were filling boundaries when they switch to outflow. Yeah, Dirichlet would be best.

Reply:
Changed diffusion BCs to Dirichlet: https://github.com/Exawind/amr-wind/pull/1066/commits/e18ba0b213bd9ca9138c53a3f68017243959e9e5